spec/lib/connection_spec.rb

```diff
@@ -147,7 +147,7 @@
       expect(auth_header == "APIAuth id123:TQiQIW6vVaDC5jvh99uTNkxIg6Q=" || auth_header == "APIAuth id123:PMWBThkB8vKbvUccHvoqu9G3eVk=").to be_truthy
     end
 
-    if Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
+    if  Gem.loaded_specs["api-auth"].present? && Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
```

Comment:
Why did this check need changing?  ApiAuth is a `development_dependency` in the .gemspec so it should always be present during RSpec runs.